presto-product-tests/conf/presto/etc/ldap-authenticator-referrals.properties

```diff
@@ -0,0 +1,5 @@
+password-authenticator.name=ldap
```

Comment:
Is this file only related to `singlenode-ldap-referrals` docker environment? If so I would store it under `singlenode-ldap-referrals` directory then.

See: https://github.com/prestosql/presto/pull/2336 and https://github.com/prestosql/presto/issues/1766

Reply:
I'm a bit confused as to the scope of changes suggested in #1766. It says:

> The idea is to restructure that so every environment had all the files it needs.
```
conf/docker/environment/compose.sh
conf/docker/environment/docker-compose.yml
conf/docker/environment/presto-master/config.properties
conf/docker/environment/presto-master/catalog/hive.properties
conf/docker/environment/hadoop-master/core-site.xml
conf/docker/environment/tempto-configuration.yaml
```

I see that #2336 created a directory for Kafka connector configuration, but for example `presto-master/etc/presto/config.properties` file seems to be missing.

Is it enough to move the `ldap-authenticator-referrals.properties` file to `conf/docker/singlenode-ldap-referrals/presto-master/etc` or should I move additional files as well?